lib/requester.rb

```diff
@@ -21,7 +21,7 @@ def parse(args)
     end
 
     def has_message?
-      !message.empty?
+      message != ""
```

Comment:
you don't like `empty?` ? :)

Reply:
I believe I had some test complaining about it - I need to retry, I think I was pretty tired when I tried out :D